app/models/agents/scheduler_agent.rb

```diff
@@ -87,7 +87,7 @@ def working?
       true
     end
 
-    def check!
```

Comment:
I can't recall what was the reason for the name, but I guess we can now remove `check!` here and make huginn_scheduler directly call `control!` instead.